<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="6">
     <layout class="QGridLayout" name="Open_Layout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Closed_Label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Closed Events From: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CalendarSelection">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Selected Date</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="Open_Display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(40, 40, 40);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="allColumnsShowFocus">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Event#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Timestamp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Location</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RP</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Phone</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Call Type / Comment</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Open_Label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Open Events:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTreeWidget" name="Closed_Display">
        <property name="minimumSize">
         <size>
          <width>567</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>280</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(40, 40, 40);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Event#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Call Type</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="Calendar_Label">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>351</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Calendar:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QCalendarWidget" name="Calendar_Dispaly">
      <property name="minimumSize">
       <size>
        <width>351</width>
        <height>171</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>351</width>
        <height>171</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calendar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);
alternate-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="Officers_Label">
      <property name="minimumSize">
       <size>
        <width>339</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>339</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Officers:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="Input_Layout">
      <item>
       <widget class="QLabel" name="Input_Label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>No Event Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Input_Edit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Input&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="Calendar_Reset">
      <property name="maximumSize">
       <size>
        <width>352</width>
        <height>29</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="DateTime">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Current Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QTreeWidget" name="Officers_Display">
      <property name="minimumSize">
       <size>
        <width>351</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>331</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>District</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Call-Sign</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
background-color: rgb(25, 25, 25);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New...</string>
     </property>
     <addaction name="actionEvent"/>
     <addaction name="actionComment"/>
     <addaction name="actionOfficer"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="actionEvent_2"/>
     <addaction name="actionDaily_Log"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="menuNew"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEvent_Type"/>
    <addaction name="actionComment_2"/>
    <addaction name="actionOfficers"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOfficers_2"/>
    <addaction name="actionCalendar"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Date"/>
    <addaction name="actionOpen_Events"/>
    <addaction name="actionClosed_Events"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionAdmin_Menu"/>
    <addaction name="actionGrant_Temp_Access"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAdmin"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionEvent">
   <property name="text">
    <string>Event</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionEvent_2">
   <property name="text">
    <string>Event</string>
   </property>
  </action>
  <action name="actionDaily_Log">
   <property name="text">
    <string>Daily Log</string>
   </property>
  </action>
  <action name="actionEvent_Type">
   <property name="text">
    <string>Event Type</string>
   </property>
  </action>
  <action name="actionComment_2">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionReset_Date">
   <property name="text">
    <string>Reset Date</string>
   </property>
  </action>
  <action name="actionOfficers">
   <property name="text">
    <string>Officers</string>
   </property>
  </action>
  <action name="actionOfficer">
   <property name="text">
    <string>Officer</string>
   </property>
  </action>
  <action name="actionOfficers_2">
   <property name="text">
    <string>Officers</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="text">
    <string>Calendar</string>
   </property>
  </action>
  <action name="actionOpen_Events">
   <property name="text">
    <string>Open Events</string>
   </property>
  </action>
  <action name="actionClosed_Events">
   <property name="text">
    <string>Closed Events</string>
   </property>
  </action>
  <action name="actionAdmin_Menu">
   <property name="text">
    <string>Admin Menu</string>
   </property>
  </action>
  <action name="actionGrant_Temp_Access">
   <property name="text">
    <string>Grant Temp Access</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
